crates/solana-event-forwarder/src/component.rs

```diff
@@ -736,8 +736,9 @@ mod tests {
                 .unwrap();
             expected_sum = expected_sum.saturating_add(tx.cost_in_lamports);
         }
+        dbg!(tx.logs);
 
-        let event_id = TxEvent::new(approve_signature.to_string().as_str(), 4);
+        let event_id = TxEvent::new(approve_signature.to_string().as_str(), 8);
```

Comment:
nit: since it's used at least twice, it could be constified somewhere to give more context on what that is, as by reading only that PR I completely don't know what it means and why we're changing that 😸 
But feel free to ignore it as it was there before you made the change 👍

Reply:
Yeah, these tests will be obsolete when we move to the new relayer. I don't think we should spend much time on it. This value is just the log line the event was found. After the changes due to audit, there are more logs and this event moved further.